crates/sovereign-sdk/full-node/db/sov-db/src/ledger_db/mod.rs

```diff
@@ -453,7 +453,23 @@ impl LedgerDB {
         Ok(())
     }
 
-    /// Used by the sequencer to record that it has committed to soft confirmations on a given L1 height
+    /// Used by the sequencer to record that it has committed to soft confirmations on a given L2 height
+    #[instrument(level = "trace", skip(self), err, ret)]
+    pub fn set_last_sequencer_commitment_l2_height(
+        &self,
+        l2_height: BatchNumber,
+    ) -> Result<(), anyhow::Error> {
+        let mut schema_batch = SchemaBatch::new();
+
+        schema_batch
+            .put::<LastSequencerCommitmentSentL2>(&(), &l2_height)
+            .unwrap();
+        self.db.write_schemas(schema_batch)?;
+
+        Ok(())
+    }
+
+    /// Used by the sequencer to record that it has committed to soft confirmations on a given L2 height
     #[instrument(level = "trace", skip(self), err, ret)]
     pub fn set_last_sequencer_commitment_l1_height(
```

Comment:
We don't need this any anymore?

Reply:
We do... for sequencer to keep track of commitments at DA slots

Reply:
sequencer is still committing when it's over with an L1 block. that's gonna change soon.